lib/vrt/mapping.rb

```diff
@@ -52,12 +56,12 @@ def load_mappings
       raise VRT::Errors::MappingNotFound if @mappings.empty?
     end
 
-    def mapping_file_path(version)
-      filename = VRT::DIR.join(version, 'mappings', "#{@scheme}.json")
+    def mapping_file_path(version, parent_dir: PARENT_DIR)
+      filename = VRT::DIR.join(version, parent_dir, "#{@scheme}.json")
       return filename if File.file?(filename)
 
       # Supports mappings that are nested under their scheme name e.g. `mappings/cvss/cvss.json`
-      VRT::DIR.join(version, 'mappings', @scheme, "#{@scheme}.json")
+      VRT::DIR.join(version, parent_dir, @scheme, "#{@scheme}.json")
```

Comment:
It seems if you just made this a class level or instance level method you could define it in this class and then just override it in the sublcasses and not need to override the initalizer in the subclass at all

Reply:
Ya definitely, I wanted to change the existing interface as little as possible, but this works too.